### DISCUSSION TITLE
Decoding signal from HAMA weather station

### DISCUSSION THREAD

**Discussion by gaspariki** [merbanan/rtl_433#3052] [Q&A]
Does anyone have experience with decoding the HAMA weather station? specifically, I have a Hama EWS Trio. Thanks

**Comment by zuckschwerdt** [comment]
If you already have the sender then preferably grab a sample check in https://triq.org/pdv/ and upload here as zip, comment the values you read for that sample.
https://triq.org/rtl_433/ANALYZE.html
